### PR TITLE
add file that was missed as part of #611

### PR DESCRIPTION
**Pull request by annaswims**
this was missing from #611

### CODE REVIEW

**Comment by timwright12** [comment]
@annaswims Looks like there are some failing tests related to this update

**Comment by annaswims** [comment]
thanks, @timwright12! I deleted the test that was no longer necessary as well

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "20.6.2",
+  "version": "20.6.4",
```
Comment (Mottie) [inline package.json:3]: version bumped twice 🤷🏽‍♂️

**Review by Mottie — APPROVED** [review]
ghe-emoji

File: package.json (hunk 1 above)
Comment (annaswims) [reply package.json:3]: I also bumped the version in https://github.com/department-of-veterans-affairs/vets-json-schema/pull/612

**Review by ericboehs — APPROVED** [review]

**Comment by timwright12** [comment]
Going to defer the review to @ericboehs as I'm not familiar with the code and it looks like you can merge without me

**Pull request merged**